src/Development/IDE/Core/RuleTypes.hs

```diff
@@ -290,6 +303,12 @@ instance Hashable GetModIface
 instance NFData   GetModIface
 instance Binary   GetModIface
 
+data GetModIfaceWithoutLinkable = GetModIfaceWithoutLinkable
```

Comment:
I don't know, this way is more consistent with `GetModIface` and `GetModIfaceWithoutTimestamps`. If you feel otherwise, we can change it. 

The only place where it makes a difference is when defining new `Rules` (since it can save recomputation if the linkable changed but the iface hash remained the same). If it is just `use`d with `runIdeAction` or something, then there shouldn't be any difference.